Mathlib/RingTheory/OreLocalization/OreSet.lean

```diff
@@ -41,36 +59,48 @@ cancellability. -/
   ore_eq : ∀ (r : R) (s : S), oreDenom r s * r = oreNum r s * s
 #align ore_localization.ore_set OreLocalization.OreSet
 
+-- TODO: use this once it's available.
+-- run_cmd to_additive.map_namespace `OreLocalization `AddOreLocalization
+
 variable {R : Type*} [Monoid R] {S : Submonoid R} [OreSet S]
 
 /-- Common factors on the right can be turned into common factors on the left, a weak form of
 cancellability. -/
+@[to_additive AddOreLocalization.ore_right_cancel]
 theorem ore_right_cancel (r₁ r₂ : R) (s : S) (h : r₁ * s = r₂ * s) : ∃ s' : S, s' * r₁ = s' * r₂ :=
   OreSet.ore_right_cancel r₁ r₂ s h
 #align ore_localization.ore_right_cancel OreLocalization.ore_right_cancel
 
 /-- The Ore numerator of a fraction. -/
+@[to_additive AddOreLocalization.oreMin "The Ore minuend of a difference."]
 def oreNum (r : R) (s : S) : R :=
   OreSet.oreNum r s
 #align ore_localization.ore_num OreLocalization.oreNum
 
 /-- The Ore denominator of a fraction. -/
+@[to_additive AddOreLocalization.oreDenom "The Ore subtrahend of a difference."]
```

Comment:
Why isn't it called something like `.oreSubtra`?

Reply:
Oops fixed.